src/main/java/uk/gov/hmcts/reform/laubackend/idam/service/UserDeletionAuditService.java

```diff
@@ -94,6 +96,14 @@ private Pageable getPage(final String size, final String page) {
         return of(parseInt(pageNumber) - 1, parseInt(pageSize), Sort.by(Sort.Direction.DESC, "deletion_timestamp"));
     }
 
+    public Pageable getPageSorted(final String size, final String page, final String sort) {
+        final String pageSize = isEmpty(size) ? defaultPageSize : size.trim();
+        final String pageNumber = isEmpty(page) ? "1" : page.trim();
+
+        return of(parseInt(pageNumber) - 1, parseInt(pageSize), Sort.by(
```

Comment:
You could extend getPage to return the records in desc order if sort parameter not provided

Reply:
Leaving this as it is